server/utils/placements/index.test.ts

```diff
@@ -139,10 +139,10 @@ describe('placementUtils', () => {
       const urlOtherId = `/manage/premises/${faker.string.uuid()}`
       expect(getBackLink(bareUrl, premisesId)).toEqual(bareUrl)
       expect(getBackLink(urlWithQuery, premisesId)).toEqual(urlWithQuery)
-      expect(getBackLink('some string', premisesId)).toEqual(bareUrl)
-      expect(getBackLink('', premisesId)).toEqual(bareUrl)
-      expect(getBackLink(null, premisesId)).toEqual(bareUrl)
-      expect(getBackLink(urlOtherId, premisesId)).toEqual(bareUrl)
+      expect(getBackLink('some string', premisesId)).toEqual(null)
+      expect(getBackLink('', premisesId)).toEqual(null)
+      expect(getBackLink(null, premisesId)).toEqual(null)
+      expect(getBackLink(urlOtherId, premisesId)).toEqual(null)
```

Comment:
As discussed - the template skips inserting the backlink if the `backlink` is null.
Also, the link from the withdrawals process opens in a new tab so adding a referrer backlink there would allow the user to carry on in the new tab - which didn't seem right.
I think we could drop the 'open in new tab' now, after consulting UCD, but we would have to implement a similar approach in legacy bookings too.